### DISCUSSION TITLE
custom 404 error for another domain

### DISCUSSION THREAD

**Discussion by ah-rahimi** [laravel/framework#44087] [General]
Hi
I want add custom 404 error for another domain/subdomain
But I can't
Please help me

**Comment by henzeb** [comment]
Read the docs?

https://laravel.com/docs/9.x/errors#custom-http-error-pages

**Reply by ah-rahimi** [reply]
Hi @henzeb 
No, I need more 404 errors 
Set a 404 error for each domain
But now I can only set one for all domains

**Reply by henzeb** [reply]
Well, blade is php. So couldn't you write some logic there?
